src/django_components/__init__.py

```diff
@@ -44,6 +45,8 @@
     "Component",
     "ComponentFileEntry",
     "ComponentFormatter",
+    "ComponentMediaInput",
+    "ComponentMediaInputPath",
```

Comment:
Exported these, so that the typings for `Media.js` and `Media.css` show up in the API reference.